extras/dagger/main.go

```diff
@@ -49,6 +49,10 @@ func (m *Chainloop) Init(
 	// Path to the source repository to be attested
 	// +optional
 	repository *Directory,
+	// Workflow name to be used for the attestation, default empty
+	// This option is needed when using an API Token
+	// +optional
+	workflowName string,
```

Comment:
I am wondering if we should make this required and basically be the only mode supported for dagger.

Reply:
Personally I would mark it as required once we have all pipelines at least migrated to the API tokens

Reply:
Why is that? Are you concerned of the feature readiness itself to do not want to confuse people with out to date documentation?

Reply:
It’s about documentation, I want the desync with it to last the bear minimum if possible